### ISSUE TITLE
Found an error in taxi_dashboard

### ISSUE THREAD

**Issue by ssackett** [jupyter/dashboards#305]
While executing the demo dashboard "taxi_dashboard" I received the error:

FileNotFoundError: File b'/home/jovyan/work/taxi_demo/datasets/fares.csv' does not exist

This caused by LOCAL_DATA being defined as:
LOCAL_DATA = '/home/jovyan/work/taxi_demo/datasets'

Instead it should point to:
./dashboards/etc/notebooks/taxi_demo/datasets

**Comment by parente** [comment]
Good find. Happy to take a pull request if you'd like to contribute a fix. Otherwise we'll get to it. I'm not sure the declarative widgets are working at the moment with latest pandas at the moment, separate from this issue.

**Comment by ssackett** [comment]
Peter,

Thanks for the help.  I'll let you know when I get things working.

Steve

**Comment by ssackett** [comment]
Peter,

Taxi_dashboard is badly broken.  The notebook runs without error but none of the graphics render in either the notebook or dashboard views.

I won't be able to offer a fix beyond what I mentioned earlier about the path to the .csv file.

Steve

**Comment by parente** [comment]
Yep. Sounds like the declarative widgets issue I mentioned. I would treat that demo as dead for the time being.
